jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml

```diff
@@ -0,0 +1,57 @@
+- job-template:
+    builders:
+    - shell: |
+        # TODO(fejta): consider a stable tag instead of master
+        git clone https://github.com/kubernetes/test-infra -b master
+        './test-infra/jenkins/bootstrap.py' --job='{job-name}' --repo='{repo-name}' --branch='{branch}' --root="${{GOPATH}}/src"
+    disabled: false
+    name: 'ci-{repo-suffix}'
+    node: 'e2e'
+    properties:
+    - build-discarder:
+        days-to-keep: 7
+    triggers:
+    - timed: '{frequency}'
+    wrappers:
+    - e2e-credentials-binding
+    - inject:
+        properties-content: |
+            GOROOT=/usr/local/go
+            GOPATH=$WORKSPACE/go
+            PATH=$PATH:$GOROOT/bin:$WORKSPACE/go/bin
+    - timeout:
+        timeout: 700
+        fail: true
+    - workspace-cleanup:
+        dirmatch: true
+        exclude:
+        - 'test-infra/.git/'
+        external-deletion-command: 'sudo rm -rf %s'
+
+- project:
+    jobs:
+    - 'ci-{repo-suffix}'
+    name: bootstrap-ci
+    repo-suffix:
+    # TODO(fejta): migrate critical jobs later
+    # kubernetes-verify-master:
+    #   branch: master
+    #   job-name: ci-kubernetes-verify-master
+    #   repo-name: k8s.io/kubernetes
+    #   frequency: 'H/5 * * * *'
+    - kubernetes-verify-release-1.5:
+        branch: release-1.5
```

Comment:
You mean between bootstrap-ci.yaml and bootstrap-ci-repo.yaml? 

The former uses `bootstrap.py --bare` since it doesn't check out a repo.
This one uses `bootstrap.py --repo=R --branch=B` since it needs the source code.